GVFS/GVFS.Virtualization/FileSystem/FileSystemVirtualizer.cs

```diff
@@ -13,6 +13,8 @@ public abstract class FileSystemVirtualizer : IDisposable
     {
         public const byte PlaceholderVersion = 1;
 
+        protected static readonly byte[] FolderContentId = Encoding.Unicode.GetBytes(GVFSConstants.AllZeroSha);

```

Comment:
Why did this say as `AllZeroSha`?

Reply:
This is what's actually stored on disk in the folder placeholders.  I wanted to avoid changing the on-disk format of the folder placeholders themselves.

Reply:
> So we never check this value and don't expect to need to tell the difference using ContentId whether a folder is partial or expanded?

That's right.  We're currently only checking the contentID of files (when we get a hydration request).

On Mac the kext uses the`FileFlags_IsEmpty` flag to check if the directory has been expanded or not.

```if (FileFlagsBitIsSet(currentVnodeFileFlags, FileFlags_IsEmpty))```

I'd rather not change this now unless we have to (to leave things cleaner for ourselves if we do need to change it some day).

Reply:
Sounds good.  Thanks!